st2api/tests/unit/controllers/v1/test_rules.py

```diff
@@ -99,8 +99,32 @@ def tearDownClass(cls):
         super(TestRuleController, cls).setUpClass()
 
     def test_get_all(self):
+        post_resp_rule_1 = self.__do_post(TestRuleController.RULE_1)
+        post_resp_rule_3 = self.__do_post(TestRuleController.RULE_3)
+
         resp = self.app.get('/v1/rules')
         self.assertEqual(resp.status_int, http_client.OK)
+        self.assertEqual(len(resp.json), 2)
+
+        self.__do_delete(self.__get_rule_id(post_resp_rule_1))
+        self.__do_delete(self.__get_rule_id(post_resp_rule_3))
+
+    def test_get_all_enabled(self):
+        post_resp_rule_1 = self.__do_post(TestRuleController.RULE_1)
+        post_resp_rule_3 = self.__do_post(TestRuleController.RULE_3)
+
+        # enabled=True
+        resp = self.app.get('/v1/rules?enabled=True')
+        self.assertEqual(resp.status_int, http_client.OK)
+        self.assertEqual(self.__get_rule_id(post_resp_rule_1), resp.json[0]['id'])
```

Comment:
It would also be a good idea to assert on the response (status = disabled / enabled, depending on the test scenario).